### DISCUSSION TITLE
How can I sync my form from local to production?

### DISCUSSION THREAD

**Discussion by handplant** [solspace/craft-express-forms#64] [Feature Requests]
### Description
I have created a form in my local environment. How do I get this form integrated with all fields in production? Are the fields not in the project.yaml and will be synced automatically?

### Additional info

- Express Forms edition & version: LITE 1.0.4
- Craft version: Craft Pro 3.3.0.1
- PHP version: 7.2.14
- Database driver & version: MySQL 5.5.5
- Other plugins & versions (if applicable):

**Comment by mtnorthrop** [comment]
I don't believe it's possible to sync forms; I always set the forms up again on production by hand. It would be really nice if there was an import/export feature.
